python/cuml/test/test_metrics.py

```diff
@@ -689,6 +693,54 @@ def test_roc_auc_score_at_limits():
         roc_auc_score(y_true, y_pred)
 
 
+def test_average_precision_recall_score():
+    y_true = np.array([0, 0, 1, 1])
+    y_pred = np.array([0.1, 0.4, 0.35, 0.8])
+    assert_almost_equal(average_precision_score(y_true, y_pred),
+                        sklearn_average_precision_score(y_true, y_pred))
+
+    y_true = np.array([0, 0, 1, 1, 0])
+    y_pred = np.array([0.8, 0.4, 0.4, 0.8, 0.8])
+    assert_almost_equal(average_precision_score(y_true, y_pred),
+                        sklearn_average_precision_score(y_true, y_pred))
+
+
+@pytest.mark.parametrize('n_samples', [50, 500000])
+@pytest.mark.parametrize('dtype', [np.int32, np.int64, np.float32, np.float64])
```

Comment:
since the consumption of different dtypes is tested as part of the test suite of the input utils, I don't think we need to do the full matrix here

Reply:
This is used overall the test_metrics methods. I was following the same standard.